src/event.js

```diff
@@ -610,8 +610,7 @@ jQuery.event = {
 			event,
 			{
 				type: type,
-				isSimulated: true,
-				originalEvent: {}
```

Comment:
This discussion seems to just keep dragging on, but as far as I can tell, nobody has provided a single reason that the property needs to, or even should exist, based on a real use case.